config/v2_2/append_test.go

```diff
@@ -20,9 +20,13 @@ import (
 	"github.com/coreos/go-semver/semver"
 	"github.com/stretchr/testify/assert"
 
-	"github.com/coreos/ignition/config/types"
+	"github.com/coreos/ignition/config/v2_2/types"
 )
 
+func intToPtr(x int) *int {
```

Comment:
Isn't there a copy of this in `internal`? Or will that cause problems with thing that want to vendor this?

Reply:
I believe that does indeed cause issues with vendors because they're outside the project space, haven't tested it locally though.

Given that we're using the `typeToPtr` functions in a few places in `config`, maybe it's worthwhile to make a centralized place to grab them from to reduce duplication though?

Reply:
I don't believe it'll cause any explicit error, but nothing in/under `config` currently depends on `internal`, and I'd prefer to not add that dependency. It's also [defined in config/translate.go](https://github.com/coreos/ignition/blob/2155a598d5ee1ebe69e66e16da6889decee1d056/config/translate.go#L30) for example. Perhaps a `config/util` package to hold all of these are in order, but I'd prefer to do that refactor in conjunction with the one that's planned to make spec stabiliziation more straightforward.